src/Kunstmaan/UtilitiesBundle/DependencyInjection/Configuration.php

```diff
@@ -18,7 +19,22 @@ class Configuration implements ConfigurationInterface
     public function getConfigTreeBuilder()
     {
         $treeBuilder = new TreeBuilder();
-        $treeBuilder->root('kunstmaan_utilities');
+        $root = $treeBuilder->root('kunstmaan_utilities');
+
+        /** @var ArrayNodeDefinition $pages */
+        $root
+            ->children()
+                ->arrayNode('cipher')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->scalarNode('secret')
+                            ->isRequired()
+                            ->cannotBeEmpty()
+                            ->defaultValue('%secret%')
```

Comment:
The default value here can be removed IMHO.

Reply:
Why remove? The clue of this change is that if you do not set a parameter value for "kunstmaan_utilities.cipher.secret", that it will use the "secret", which is already defined..

Reply:
Yes that's true, but that value is being set in your if else statement in the KunstmaanUtilitiesExtension.php, right?

Reply:
Yes, true.

Reply:
So we don't need to provide it here then?